src/mini_components/react_component.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const name = 'Vinodel'
```

Comment:
Based on the way you use this component in App, this should not be a variable, rather a prop.